plasmapy/diagnostics/langmuir.py

```diff
@@ -22,29 +23,30 @@ class Probe():
         Electrode configuration. Possible options are 'single' (default),
         'double', 'double-asym', 'triple', 'tetra', 'penta'. The electrode
         configuration must correspond to the number of areas provided.
-    
+
     """
     def __init__(self, area=u.m**2, config='single'):
         # Initialize probe properties
         self.area = area
         self.config = config
```

Comment:
Coveralls mentions that these two lines don't get tested. Probably not so important now, but will be important once you start writing tests.